frontend/packages/knative-plugin/src/utils/knative-topology-utils.ts

```diff
@@ -110,7 +109,8 @@ export const getKnativeTopologyNodeItem = (
   resource: K8sResourceKind,
   type: string,
   resources?: TopologyDataResources,
-): Node => {
+): Node[] => {
```

Comment:
This function is called `getKnativeTopologyNodeItem`, returning an array of items is confusing API.